test/PlaygroundTransform/Inputs/not.py

```diff
@@ -0,0 +1,15 @@
+import sys
+import subprocess
+
+if len(sys.argv) < 2:
+    print("Too few args to " + sys.argv[0])
+    sys.exit(0)
+
+try:
+    cmd = sys.argv[1]
+    print(subprocess.check_call(cmd, stderr=subprocess.STDOUT, shell=True))
```

Comment:
The `stderr=subprocess.STDOUT` isn't necessary either since you still have `2>&1` in the invocation. You can also avoid `shell=True` by *not* passing the arguments as a single string, which is probably more portable.